src/main/java/Uranus/Tasks/Deadlines.java

```diff
@@ -0,0 +1,27 @@
+package Uranus.Tasks;
+
+public class Deadlines extends Task{
+
+    protected String by;
+
+    public Deadlines(String description){
+        super(description, "D");
+    }
+
+    public void setBy(String by){
+        this.by = by;
+    }
+
+    public String getBy(){
+        return by;
+    }
+
+    @Override
+    public void setDescription(String description){
+        int separatorIndex = description.indexOf(' ');
+        String[] str = description.split("/by");
```

Comment:
Is it possible to remove the use of magic numbers and strings in the entire codebase?